concepticondata/conceptlists/Wilson-2015-173.tsv

```diff
@@ -0,0 +1,174 @@
+Wilson-2015-173-137	137	TO SIT	sedeo	sentar	to sit	1	Verb	11	1,25	0,03	117,21	1416	SIT
+Wilson-2015-173-138	138	TO SLEEP	dormio	dormir	to sleep	0	Verb	15	2,01	0,03	142,21	1585	SLEEP
+Wilson-2015-173-139	139	TO SMELL (PERCEIVE ODOR)	oleo	oler	to smell (perceive odor)	0	Verb	36	6,48	0,04	31,32	1586	SMELL (PERCEIVE)
+Wilson-2015-173-140	140	TO SPIT	spuo	escupir	to spit	4	Verb	17	2,04	0,03	9,85	1440	SPIT
+Wilson-2015-173-141	141	TO SPLIT	findo	hender	to split	1	Verb	31	5,58	0,04	140,59	1437	SPLIT
+Wilson-2015-173-142	142	TO SQUEEZE	exprimo	exprimir	to squeeze	0	Verb	42	7,83	0,04	25,29	1414	SQUEEZE
+Wilson-2015-173-143	143	TO SUCK	sugo	mamar	to suck	4	Verb	19	2,41	0,03	13,38	1421	SUCK
+Wilson-2015-173-144	144	TO SWIM	nato	nadar	to swim	0	Verb	22	2,73	0,03	13,97	1439	SWIM
+Wilson-2015-173-145	145	TO THINK	cogito	pensar	to think	4	Verb	36	5,34	0,03	710	2271	THINK
+Wilson-2015-173-146	146	TO THROW	iacio	lanzar	to throw	4	Verb	42	7,52	0,03	88,38	1456	THROW
+Wilson-2015-173-147	147	TO TIE	ligo	atar	to tie	4	Verb	21	2,68	0,03	22,06	1917	TIE
+Wilson-2015-173-148	148	TO TURN (VEER)	gyro	volver	to turn (veer)	4	Verb	34	8,18	0,05	8,82	1444	TURN AROUND
+Wilson-2015-173-149	149	TO VOMIT	vomo	vomitar	to vomit	1	Verb	32	5,02	0,03	9,56	1278	VOMIT
+Wilson-2015-173-150	150	TO WALK	ambulo	andar	to walk	4	Verb	36	5,14	0,03	232,65	1443	WALK
+Wilson-2015-173-151	151	TO WASH	lavo	lavar	to wash	0	Verb	24	3	0,03	38,24	1453	WASH
+Wilson-2015-173-152	152	TONGUE	lingua	lengua	tongue	1	Noun	4	0,49	0,06	116,47	1205	TONGUE
+Wilson-2015-173-153	153	TOOTH	dens	diente	tooth	1	Noun	7	1,18	0,05	26,03	1380	TOOTH
+Wilson-2015-173-154	154	TREE	arbor	árbol	tree	2	Noun	26	3,58	0,03	67,94	906	TREE
+Wilson-2015-173-155	155	TWO	duo	dos	two	2	Number	1	0,09	0,09	1412,94	1498	TWO
+Wilson-2015-173-156	156	WARM (WEATHER)	calidus	cálido	warm (weather)	1	Adj.	16	2,35	0,04	32,35	2110	WARM (OF WEATHER)
+Wilson-2015-173-157	157	WATER	aqua	agua	water	1	Adj.	8	0,86	0,04	241,62	948	WATER
+Wilson-2015-173-158	158	WE	nos	nosotros	we	3	Pronoun	2	0,37	0,12	1097,06	1212	WE
+Wilson-2015-173-159	159	WET	umidus	húmedo	wet	0	Adj.	28	4,45	0,03	10,29	1726	WET
+Wilson-2015-173-160	160	WHEN	quando	cuando	when	0	Adv.	10	1,08	0,03	1899,71	1238	WHEN
+Wilson-2015-173-161	161	WHERE	ubi	donde	where	3	Adv.	7	0,7	0,04	884,85	1237	WHERE
+Wilson-2015-173-162	162	WHITE	albus	blanco	white	5	Adj.	17	2,22	0,03	121,62	1335	WHITE
+Wilson-2015-173-163	163	WHO	qui	quién	who	1	Pronoun	1	0,09	0,09	452,35	1235	WHO
+Wilson-2015-173-164	164	WIDE	latus	ancho	wide	4	Adj.	20	2,89	0,03	18,97	1243	WIDE
+Wilson-2015-173-165	165	WIFE	uxor	esposa	wife	4	Noun	31	4,12	0,03	62,5	1199	WIFE
+Wilson-2015-173-166	166	WIND (BREEZE)	ventus	viento	wind (breeze)	1	Noun	16	1,78	0,03	47,79	960	WIND
+Wilson-2015-173-167	167	WING	ala	ala	wing	1	Noun	22	2,66	0,03	23,97	1257	WING
+Wilson-2015-173-168	168	WOMAN	mulier	mujer	woman	0	Noun	25	2,75	0,02	591,62	962	WOMAN
+Wilson-2015-173-169	169	WOODS	silva	selva	woods	1	Noun	27	4,15	0,03	43,53	420	FOREST
```

Comment:
Is it the same as if we mapped it as `2311 WOODS`? Also, the Spanish translation would be mapped as `3808 JUNGLE`, while the latin as `420 FOREST`.

Reply:
WOODS is also a duplicate. @MuffinLinwist You can check if a concept has a duplicate by looking at the REPLACEMENT_ID column in concepticon.tsv. If there is an ID listed there, it means that the concept has been replaced by the initial concept.

Reply:
Regarding the FOREST / JUNGLE suggestion. We have a Portuguese list with _selva_ mapped to JUNGLE and two Spanish lists with _selva_ mapped to FOREST. Is this distinction correct or should all of them be mapped to JUNGLE?

Reply:
@AnnikaTjuka, I believe, according to the definitions on Conception, JUNGLE would be more suitable for both languages.

Reply:
Ok, thanks! @mathildavz Could you remap the gloss to JUNGLE and open an issue that we also need to remap the Spanish lists?

Reply:
Of course! @AnnikaTjuka